app/assets/stylesheets/typography.scss

```diff
@@ -1,7 +1,40 @@
-body,
-button,
-input,
-select,
-textarea {
-  font-family: "Clear Sans", "Segoe UI", Helvetica, Arial, sans-serif;
+$font-stack: "Clear Sans", "Segoe UI", Helvetica, Arial, sans-serif;
```

Comment:
If you have thoughts on other fallback fonts, I just kind of threw these in there; I'm not particularly attached to them.